src/arrayMethodSort.js

```diff
@@ -4,8 +4,21 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (
+    compareFunction = (a, b) => (String(a) > String(b) ? 1 : -1),
```

Comment:
The default comparison function uses lexicographical comparison, which is not suitable for numbers. Consider changing it to `(a, b) => a - b` for numerical sorting by default.